src/OrbitCaptureClient/CaptureClient.cpp

```diff
@@ -58,6 +58,7 @@ static CaptureOptions::InstrumentedFunction::FunctionType InstrumentedFunctionTy
         FunctionInfo_OrbitType_FunctionInfo_OrbitType_INT_MAX_SENTINEL_DO_NOT_USE_:
       UNREACHABLE();
   }
+  UNREACHABLE();
```

Comment:
Just out of curiosity, why is this necessary? Did a build fail?

Reply:
Some checks failed with the error of "control reaches the end of a non-void function".

![Screenshot 2021-03-18 132908](https://user-images.githubusercontent.com/74234352/111626178-9c089200-87e5-11eb-8b33-5e1de6d7c606.png)